http-filter-example/http_filter.cc

```diff
@@ -0,0 +1,87 @@
+#include <string>
+
+#include "common/common/base64.h"
+#include "common/common/logger.h"
+#include "common/http/headers.h"
+#include "common/http/utility.h"
+#include "envoy/server/instance.h"
+#include "envoy/ssl/connection.h"
+#include "server/config/network/http_connection_manager.h"
+
+
+namespace Envoy{
+namespace Http{
+namespace SampleFilter{
+
+class Config {
+ public:
+  Config(const Json::Object&) {}
+};
+typedef std::shared_ptr<Config> ConfigPtr;
+
+const LowerCaseString headerKey = LowerCaseString("via");
+const std::string headerValue = "sample-filter";
+
+class Instance : public Http::StreamDecoderFilter {
```

Comment:
You mean "Sample" Filter? Or, should I also change the filter's name used in the config to "simple"?